.changeset/purple-cats-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': minor
+---
+
+`GovernorTimelockControl`: Add the Governor instance address as part of the TimelockController operation `salt` to avoid operation id collisions between governors using the same TimelockController.
```

Comment:
@ernestognw I added a period here. Please include punctuation in changesets in general.

Reply:
Sounds good, thanks!